docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -9,7 +9,7 @@ tags:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-To decrease the database size when using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage format, enable the early access feature `--Xbonsai-limit-trie-logs-enabled`. 
+Since Besu version 24.6.0, when using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage format, [`--bonsai-limit-trie-logs-enabled`](../reference/cli/options.md#bonsai-limit-trie-logs-enabled) is enabled by default, unless [`--sync-mode=FULL`](../reference/cli/options.md#sync-mode) in which case this option is disallowed and must be set to false.
```

Comment:
I don't think we have to include the version; maybe we just add it to the version if it crucial to mention. It most likely is also in the changeling. 

```suggestion
When using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage format, [`--bonsai-limit-trie-logs-enabled`](../reference/cli/options.md#bonsai-limit-trie-logs-enabled) is enabled by default. 
If [`--sync-mode=FULL`](../reference/cli/options.md#sync-mode) is set,  this option is disallowed and must be set to false.
```

Reply:
So, as per Joan's recommendation, add the new docs to:

- `/docs/public-networks/how-to/bonsai-limit-trie-logs.md`
- `/docs/public-networks/reference/cli/options.md`
- `/versioned_docs/version-24.6.0/public-networks/how-to/bonsai-limit-trie-logs.md`
- `/versioned_docs/version-24.6.0/public-networks/reference/cli/options.md`

And add a note or prerequisite confirming the user is on the correct version.

@siladu let us know if this makes sense to you and we can make the appropriate changes.

Reply:
Yep, that makes sense :)